h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.component.ts

```diff
@@ -245,7 +245,7 @@ export class ComputeCapacityComponent implements OnInit {
         results['endpointCpu'] = this.form.get('endpointCpu').value;
         results['endpointMemory'] = this.form.get('endpointMemory').value;
       }
-      return Observable.of({ computeCapacity: results });
```

Comment:
Won't this changes break the CLI command which is autogenerated based on these property names?

Reply:
That's something that we need to check since this is WIP. Please make any adjustments necessary.